Package.swift

```diff
@@ -21,7 +21,7 @@
 import PackageDescription
 
 let package = Package(
-    name: "ArrowSwift",
+    name: "Arrow",
```

Comment:
Is Arrow already taken? https://swiftpackageindex.com/freshOS/Arrow

Reply:
We use different organizations ("freshOS" and "apache" https://swiftpackageindex.com/apache ). This will not be conflicted.

Reply:
Nice, glad they chose to namespace.